vars/devnetDeploy.groovy

```diff
@@ -2,27 +2,37 @@
 import org.jenkinsci.plugins.workflow.support.steps.build.RunWrapper
 
 void call(Map config = [:]) {
-    String devnetDeployJob = '/private/cd/Devnet deploy'
+    String devnetDeployJob = '/private/cd/Deployments/Veganet/Devnet'
     Boolean wait = config.wait ? "${config.wait}".toBoolean() : false
     Boolean ignoreFailure = config.ignoreFailure ? "${config.ignoreFailure}".toBoolean() : false
     String restart = pipelineDefaults.dev.restart
     if (config.restart == true) {
-        restart = pipelineDefaults.restartOptions.restartOnly
+        restart = 'YES'
     } else if (config.restart == false) {
-        restart = pipelineDefaults.restartOptions.dontRestart
+        restart = 'NO'
     } else if (config.restart == 'checkpoint' || config.checkpoint == true) {
-        restart = pipelineDefaults.restartOptions.restartFromCheckpoint
```

Comment:
just Q: Why We do not use `pipelineDefaults.restartOptions.restartFromCheckpoint`? It will be harder to refactor when someone change "YES" to `True` or something. 


When we have some constant value we can refactor, and review it easier.

Reply:
I haven't figured out how to easily connect Jenkins Lib & DSL code. Most probably I will do later some refactor that will depend on YAML file that could be read by DSL and accessed by Jenkins method. They are kind of separated technologies